### PR TITLE
added ReflectionProperty::getDefaultValueAsString()

### PR DESCRIPTION
**Pull request by vlastavesely**

### CODE REVIEW

**Comment by asgrim** [comment]
I'm not sure why the addition of `ValueDumper` is useful. It's called statically so is in reality no better than just using `var_export` directly in each location... Just introducing unnecessary complexity IMO, which not keen on. Is there a specific reason this is useful please?

**Comment by vlastavesely** [comment]
I can imagine one. In the future you can decide to replace, for example, `var_export`'s `array()` to `[]`. If it is wrapped in the dumper, it is easy, when it would be on many places (I can imagine class constants also), it would be a little bit harder to keep it all the same.

**Comment by asgrim** [comment]
I don't think that's really useful here, I'm afraid :/ the new endpoint is fine but can we just use `var_export` directly

**Comment by vlastavesely** [comment]
So should it be removed?

File: src/Reflection/ReflectionParameter.php [hunk 1]
```diff
@@ -317,7 +318,7 @@ public function getDefaultValue()
      */
     public function getDefaultValueAsString() : string
     {
-        return var_export($this->getDefaultValue(), true);
+        return ValueDumper::dump($this->getDefaultValue());
```
Comment (Ocramius) [inline src/Reflection/ReflectionParameter.php]: `ValueDumper` is not needed here: `var_export()` is portable, stable, security-checked and does exactly what is needed here

File: src/Reflection/ReflectionParameter.php [hunk 2]
```diff
@@ -317,7 +318,7 @@ public function getDefaultValue()
      */
     public function getDefaultValueAsString() : string
```
Comment (Ocramius) [inline src/Reflection/ReflectionParameter.php]: @asgrim why do we even have this API? :-\

File: src/Util/ValueDumper.php [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Roave\BetterReflection\Util;
+
+/**
+ * @internal
+ */
+final class ValueDumper
```
Comment (Ocramius) [inline src/Util/ValueDumper.php]: This class is not really needed - the static coupling is there anyway

**Comment by vlastavesely** [comment]
requested changes already pushed.

**Comment by asgrim** [comment]
Thank you @vlastavesely :)

**Pull request merged**